int64 tick_id
int64 bee_id
int64 action
int64 target_flower_id
